src/lang/russian.txt

```diff
@@ -5357,3 +5359,36 @@ STR_PLANE                                                       :{BLACK}{PLANE}
 STR_SHIP                                                        :{BLACK}{SHIP}
 
 STR_TOOLBAR_RAILTYPE_VELOCITY                                   :{STRING} ({VELOCITY})
+
+STR_BUILD_VEHICLE_SMART                                         :Умное меню покупки техники
```

Comment:
Even if it's your native language, we prefer if translations for new strings are not added in the PR/patch introducing the new feature. They should be added after merge with the web-translator tool.